data/sql/updates/pending_db_world/rev_1636206943540376100.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1636206943540376100');
+
+ALTER TABLE `updates` CHANGE `state` `state` ENUM('RELEASED','CUSTOM','MODULE','ARCHIVED') CHARSET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT 'RELEASED' NOT NULL COMMENT 'defines if an update is released or archived.';
```

Comment:
```suggestion
ALTER TABLE `updates` CHANGE `state` `state` ENUM('RELEASED','CUSTOM','MODULE','ARCHIVED') CHARSET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT 'RELEASED' NOT NULL COMMENT 'defines if an update is released, archived, custom or from a module.';
```

Tbh Module and custom is the same thing. But as I know you didn't implement it I will just suggest the comment update :D

Reply:
@Kitzunu I'm just fixing the messup from here: https://github.com/azerothcore/azerothcore-wotlk/pull/8927#issuecomment-962452849 so I had to readd those files to fix the revision/db import script

@UltraNix could better address your suggestion

Reply:
Module and custom is now different.

Reply:
ok